packages/agoric-cli/tools/resm-plugin/src/output.js

```diff
@@ -1 +1,3 @@
+// @jessie-check
```

Comment:
I'm not confident that this is benign. I'm not sure how this resm-plugin works, but the name `output.js` suggests that it's suppose to match something, and tweaking it this way might disturb that.

Lack of a test suggests that we should be able to make this tweak, but I'm inclined to be conservative at this point in the release.